SEO: reword introduction paragraph into descriptions - Part 1 (#408)
front-commerce/developers.front-commerce.com: source/docs/advanced/theme/translations.md

---
id: translations
title: Translate your application

> description:
>   International e-commerce websites often need to translate their UI in several languages to make sure that they reach a broader range of customers.
>   In this documentation, we will see how this works for static content in your application.

Reviewer: Are line breaks supported in the description? if yes then this can be ignored (you can apply use the second suggestion)

### without linebreaks
```suggestion
description: International e-commerce websites often need to translate their UI into several languages to make sure that they reach a broader range of customers. This documentation will show how this works for static content in your application.
```

### with linebreaks
```suggestion
description:
  International e-commerce websites often need to translate their UI into several languages to make sure that they reach a broader range of customers.
  This documentation will show how this works for static content in your application.
```
